packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthenticationHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled'
+import {Info} from '@mui/icons-material'
+import React from 'react'
+import DialogTitle from '../../../../components/DialogTitle'
+import {PALETTE} from '../../../../styles/paletteV3'
+import makeMaxWidthMediaQuery from '../../../../utils/makeMaxWidthMediaQuery'
+
+const mobileBreakpoint = makeMaxWidthMediaQuery(420)
+
+const Inner = styled('div')({
+  margin: '0 auto',
+  maxWidth: '608px',
+  padding: '24px 16px 28px 16px',
+  [mobileBreakpoint]: {
```

Comment:
-1 Paddings are acting a bit weird, we add a fixed padding on width 420, but on a larger width they are dynamic. 

421 px
<img width="343" alt="image" src="https://user-images.githubusercontent.com/466991/218968472-7b7b4e8c-3889-4926-8653-a50efce828ca.png">

420 px

<img width="337" alt="image" src="https://user-images.githubusercontent.com/466991/218968562-61ae7192-4351-4001-bcfa-dcfbeb1b0021.png">

That is because instead of paddings we use:

```
margin: '0 auto',
  maxWidth: '608px',
  ```
  Which looks like paddings but instead just center the max-width block

If we always need a fixed left padding, lets remove margin, maxWidth and mobileBreakpoint and just set this css:

```
padding: 24px 16px 28px 34px;
```

Where `34px` is a left padding we need (put desired number here)